components/Form/index.js

```diff
@@ -0,0 +1,332 @@
+                {" "}
+                <label htmtFor="gynecologist">
+                  {" "}
+                  <StyledCheckbox
+                    type="checkbox"
+                    id="gynecologist"
+                    name="filterCriteria"
+                    value="gynecologist"
+                  />
+                  Gynäkolog*in
+                </label>
+              </div>
+            </StyledSection>
+          </fieldset>
+
+          <fieldset>
+            {" "}
+            <label htmlFor="target">
+              Zielgruppe:
+              <StyledInput type="text" id="target" name="target" required />
+            </label>
+          </fieldset>
+          <fieldset>
+            <label htmlFor="requirements">
+              Zugangsvoraussetzungen:
+              <StyledInput
+                type="text"
+                id="requirements"
+                name="requirements"
+                required
+              />{" "}
+            </label>
+          </fieldset>
+          {/* <label htmlFor="image">Logo:</label>
+        <input
+          type="image"
+          id="image"
+          src=""
+          alt="HandsWithaHeart"
+          width="48"
+          height="48"
+        /> */}
+          <fieldset>
+            {" "}
+            <label htmlFor="adress">
+              Adresse:
+              <StyledInput type="text" id="adress" name="adress" required />
+            </label>
+          </fieldset>
+          <fieldset>
+            {" "}
+            <label htmlFor="link">
+              Google-Maps-Link:
+              <StyledInput
+                type="url"
+                id="link"
+                name="link"
+                placeholder="https://www.google.com/maps/..."
+                pattern="https://www.google.com/maps/.*"
+                required
+              />
+            </label>
+          </fieldset>
+          <fieldset>
+            <label htmlFor="tel">
+              Telefonnummer:{" "}
+              <StyledInput
+                type="number"
+                id="tel"
+                name="tel"
+                placeholder="0049..."
+                pattern="0049"
+                required
+              />
+            </label>
+          </fieldset>
+          <button type="submit" value="submit">
+            Hinzufügen
+          </button>
+        </fieldset>
+      </StyledForm>
+    </>
+  );
+}
+
+const StyledForm = styled.form`
+  display: flex;
+  flex-direction: column;
+`;
+
+const StyledSection = styled.section`
+  display: flex;
+  justify-content: space-between;
+  ${({ variant }) =>
+    variant === "show" &&
+    css`
+      size: 20px;
+    `}
+  ${({ variant }) =>
+    variant === "hide" &&
+    css`
+      display: none;
+    `}
+`;
+
+const StyledInput = styled.input`
+  size: 30px;
+  margin: 8px 0;
+  cursor: text;
+`;
+const StyledCheckbox = styled(StyledInput)`
+  size: 20px;
+  padding: 0;
+  width: 20%;
+`;
+const StyledSvg = styled.svg``;
```

Comment:
As this does not include any styles, it seems like you do not need it. So you should erase it 🧹 
```suggestion

```